paf-mvp-frontend/cypress/integration/welcome-widget.spec.ts

```diff
@@ -72,12 +72,9 @@ describe('Welcome widget view', () => {
     it('should call the "new id" endpoint', () => {
       cy.window().then((win) => {
         const NEW_ID = 'NEW-USER-ID';
-        const response = {
-          body: { identifiers: getFakeIdentifiers(NEW_ID) },
-        };
-        cy.intercept('GET', '/paf-proxy/v1/new-id', { body: response }).as('newId');
+        const getNewIdSpy = cy.stub(win.PAF, 'getNewId').returns(Promise.resolve(getFakeIdentifiers(NEW_ID)[0]));
```

Comment:
I suppose this file should not be modified. Merge issue?

Reply:
Correct. Suspect it came from main in the past. Once review is complete perhaps we can eliminate files that were not changed in this PR.